<!DOCTYPE html>
<html lang='en'>


<head>
    <base href="..">
    <link rel="shortcut icon" type="image/png" href="assets/favicon.png"/>
    <link rel="stylesheet" type="text/css" media="all" href="assets/main.css"/>
    <meta name="description" content="Statistics Conference in Genomics, Pharmaceutical Science, and Health Data Science">
    <meta name="resource-type" content="document">
    <meta name="distribution" content="global">
    <meta name="KeyWords" content="Conference">
    <title>Program | Statistics Conference in Genomics, Pharmaceutical Science, and Health Data Science</title>
</head>


<body>

   <div class="banner">
        <img src="assets/banner.png" alt="Conference Template Banner" style="width: 98%">
        <div class="top-left">
<!--            <span class="title3">Statistics Conference in Genomics, Pharmaceutical Science, and Health Data Science </span> <!--<span class="title2">Template</span> <span class="year">2525</span> -->
        Statistics Conference in Genomics, Pharmaceutical Science, and Health Data Science 
        </div>
        <div class="bottom-right">
            August 15-17, 2022 <br> University of Victoria, Victoria, BC, Canada
        </div>
    </div>

    <table class="navigation">
        <tr>
            <td class="navigation">
                <a title="Conference Home Page" href=".">Home</a>
            </td>
            <td class="navigation">
                <a title="Register for the Conference" href="registration">Registration</a>
            </td>
            <td class="navigation">
                <a class="current" title="Conference Program" href="program">Program</a> 
            </td>
            <td class="navigation">
                <a title="Directions to the Conference" href="directions">Directions</a>
            </td>
            <td class="navigation">
                <a title="Conference Announcements" href="flyer">Announcements</a>
            </td>
        </tr>
    </table>

<!---
    <h2>Tentative Speakers (Ordered by Last Name)</h2>

    <style>
      ul {
        list-style-type: square;
        color:#999999;
      }
      ul li:before {
        content: '\2014';
        position: absolute;
        margin-left: -20px;
      }
    </style>

    <ul>
    	<li>Jiguo Cao - Simon Fraser University - Department of Statistics and Actuarial Science</li>
    	<li>Cong Chen - Merck & Co. Inc. (USA)</li>
    	<li>Xuanjin Cheng - Canada's Michael Smith Genome Sciences Cenlie (GSC) at BC Cancer</li>
    	<li>Gabriela V. Cohen Freue - University of British Columbia - Department of Statistics</li>
    	<li>Ehsan Karim - University of British Columbia - School of Population and Public Health</li>
    	<li>Mary Lesperance - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Longhai Li - University of Saskatchewan - Department of Mathematics and Statistics</li>
    	<li>Yang Liu - Veracyte, Inc.</li>
    	<li>Elham Majd - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Michelle Miranda - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Farouk Nathoo - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Ibrahim Numanagic - University of Victoria - Department of Engineering and Computer Science</li>
    	<li>Haolun Shi - Simon Fraser University - Department of Statistics and Actuarial Science</li>
    	<li>Jabed Tomal - Thompson Riveres University - Department of Statistics</li>
    	<li>Liangliang Wang - Simon Fraser University - Department of Statistics and Actuarial Science</li>
    	<li>Lang Wu - University of British Columbia - Department of Statistics</li>
    	<li>Li Xing - University of Saskatchewan - Department of Mathematics and Statistics</li>
    	<li>Shentu Yue - Daiichi Sankyo, Inc. (USA)</li>
    	<li>Xuekui Zhang - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Julie Zhou - University of Victoria - Department of Mathematics and Statistics</li>
    </ul>
--->



    <h2>Conference Program</h2>
<tr>
(Each talk would be 30-minute long with 10-minute question time.)
</tr>
<br></br>
<!---
    <table>

        <tr>
            <td class="date" rowspan="2">
                8:30am
            </td>
            <td class="title-special">
                Registration and Coffee &amp; Tea!
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Here&rsquo;s a section in the program that isn&rsquo;t a talk. 
                Notice that the title is styled differently than the ones for talks.
            </td>
        </tr>
    </table>
--->
<br>
<div style="width: 100%; height: 10px; border-bottom: 1px solid black; text-align: center">
  <span style="font-size: 24px; background-color: #fafafa; padding: 0 10px;">
    August 15<!--Padding is optional-->
  </span>
</div>
<br>

    <table>
        <tr>
            <td class="date" rowspan="2">
                9:20 am
            </td>
            <td class="title-special">
                Light Breakfast
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Breakfast provided for everyone. Located in the conference room.
            </td>
        </tr>
    </table>

    <table id="JiguoCao">
        <tr>
            <td class="date" rowspan="3">
		9:45 am
            </td>
            <td class="title">
                Predicting the onset of breast cancer using mammogram imaging data with irregular boundary
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Jiguo Cao (Simon Fraser University - Department of Statistics and Actuarial Science)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                With mammography being the primary breast cancer screening strategy, it is essential to make full use of mammogram imaging data to better identify women who are at higher and lower than average risk. Our primary goal in this study is to extract mammogram-based features that augment the well-established breast cancer risk factors to improve prediction accuracy. In this article, we propose a supervised functional principal component analysis (sFPCA) over triangulations method for extracting features that are ordered by the magnitude of association with the failure time outcome. The proposed method accommodates the irregular boundary issue posed by the breast area within the mammogram imaging data with flexible bivariate splines over triangulations. We also provide an eigenvalue decomposition algorithm that is computationally efficient. Compared to the conventional unsupervised FPCA method, the proposed method results in a lower Brier Score and higher area under the ROC curve (AUC) in simulation studies. We apply our method to data from the Joanne Knight Breast Health Cohort at Siteman Cancer Center. Our approach not only obtains the best prediction performance compared to unsupervised FPCA and benchmark models but also reveals important risk patterns within the mammogram images. This demonstrates the importance of utilizing additional supervised image-based features to clarify breast cancer risk.
	    </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
            10:25 am
            </td>
            <td class="title">
                Online Bayesian phylogenetic inference via sequential Monte Carlo 
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Liangliang Wang (Simon Fraser University - Department of Statistics and Actuarial Science)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Bayesian phylogenetics is to approximate a posterior distribution of phylogenetic trees based on biological sequence data such as DNA and RNA. Modern technologies can generate a large amount of sequence data every day. Most existing methods in Bayesian phylogenetics are costly to conduct inference for streaming data because they must start the computation anew each time when a new sequence is available. In this work, we propose an efficient online Bayesian phylogenetic method which can update an existing posterior with new sequences.  The proposed method is based on a sequential Monte Carlo method with a novel guided proposal distribution. 
            </td>
        </tr>
    </table>

    <table>

        <tr>
            <td class="date" rowspan="2">
                11:05 am
            </td>
            <td class="title-special">
                Coffee Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Coffee break provided for everyone.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                11:20 am
            </td>
            <td class="title">
                Optimization in Pharmacogenomics
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Ibrahim Numanagic (University of Victoria - Department of Engineering and Computer Science)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                High-throughput sequencing provides the means to determine allelic decomposition— the exact sequence content of all gene copies present in the sample, also known as haplotype— for any gene of interest. When applied to pharmaceutical genes, such decomposition can be used to inform the drug treatment and dosage decisions. However, many clinically and functionally important genes are highly polymorphic and have undergone structural alterations, and as such present a significant challenge for the existing genotyping methods. Here we present a combinatorial optimization framework based on integer linear programming that is able to efficiently solve this problem for various pharmacogenes, including those with structural alterations. We also show how to adapt these linear programs for the emerging long reads sequencing datasets.
            </td>
        </tr>
    </table>
    <table>

        <tr>
            <td class="date" rowspan="2">
                12:00 - 2:00 pm
            </td>
            <td class="title-special">
                Lunch Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Lunch provided for speakers and volunteers. Located in David Turpin Building first floor hallway.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                2:00 pm
            </td>
            <td class="title">
                EPPS: a Novel Ensemble Test to Improve the Power of Genomic Studies
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Xuekui Zhang (University of Victoria - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
         The traditional SNP-wise test for Genome-Wide Association Studies (GWASs) often involves testing associations between a disease outcome and millions of single nucleotide polymorphisms (SNPs) and applies multiple testing adjustments afterwards to control the false positive rate. Due to the curse of dimensionality and the limited sample size, many GWASs face a lack of power issues based on such an approach. We proposed EPPS, a novel ensemble test procedure to increase the power of GWASs. Using the multiple random data splits approach, EPPS is an ensemble two-step test procedure inspired by the ensemble machine learning approach and a traditional pilot study in genomic experiments. EPPS provides `one' p-value for each SNP by integrating results of all data splits, enabling FDR control using any standard multiple testing adjustment approach. Furthermore, automatically selects values of its parameters to optimize study power with a pre-hoc power analysis.            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                2:40 pm
            </td>
            <td class="title">
                Subtype Analysis with Cancer Somatic Mutations 
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Chad (Qianchuan) He (Fred Hutch - Public Health Sciences Division)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Understanding the association between cancers subtypes and genetic variations is fundamental to the development of targeted therapies for patients. Somatic mutation plays important roles in tumor development and has emerged as a new type of genetic variations for studying the association with cancer subtypes. We propose an approach, SASOM, for the association analysis of cancer subtypes with somatic mutations. Our approach tests the association between a set of somatic mutations (from a genetic pathway) and subtypes, while incorporating functional information of the mutations into the analysis. In a real data application, we examine the somatic mutations from a cutaneous melanoma dataset, and identify a genetic pathway that is associated with immune-related subtypes.
            </td>
        </tr>
    </table>

    <table>

        <tr>
            <td class="date" rowspan="2">
                3:20 pm
            </td>
            <td class="title-special">
                Coffee Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Coffee break provided for everyone.
            </td>
        </tr>
    </table>

    <table id="XuanjinCheng">
        <tr>
            <td class="date" rowspan="3">
                3:50 pm
            </td>
            <td class="title">
                cSurvival: a web resource for biomarker interactions in cancer outcomes and in cell lines
            </td>
        </tr>
        <tr>
            <td class="speaker"  style="font-size: 19px">
                Xuanjin Cheng (Canada's Michael Smith Genome Sciences Centre [GSC] at BC Cancer)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Survival analysis is a technique for identifying prognostic biomarkers and genetic vulnerabilities in cancer studies. Large-scale consortium-based projects have profiled >11 000 adult and >4000 pediatric tumor cases with clinical outcomes and multiomics approaches. This provides a resource for investigating molecular-level cancer etiologies using clinical correlations. Although cancers often arise from multiple genetic vulnerabilities and have deregulated gene sets (GSs), existing survival analysis protocols can report only on individual genes. Additionally, there is no systematic method to connect clinical outcomes with experimental (cell line) data. To address these gaps, we developed cSurvival (https://tau.cmmt.ubc.ca/cSurvival). cSurvival provides a user-adjustable analytical pipeline with a curated, integrated database and offers three main advances: (i) joint analysis with two genomic predictors to identify interacting biomarkers, including new algorithms to identify optimal cutoffs for two continuous predictors; (ii) survival analysis not only at the gene, but also the GS level; and (iii) integration of clinical and experimental cell line studies to generate synergistic biological insights. To demonstrate these advances, we report three case studies. We confirmed findings of autophagy-dependent survival in colorectal cancers and of synergistic negative effects between high expression of SLC7A11 and SLC2A1 on outcomes in several cancers. We further used cSurvival to identify high expression of the Nrf2-antioxidant response element pathway as a main indicator for lung cancer prognosis and for cellular resistance to oxidative stress-inducing drugs. Altogether, these analyses demonstrate cSurvival’s ability to support biomarker prognosis and interaction analysis via gene- and GS-level approaches and to integrate clinical and experimental biomedical studies.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                4:30 pm
            </td>
            <td class="title">
                Overview of Decipher/Veracyte Efforts in Building Genomic Classifiers via Semi-supervised Machine Learning Approaches
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Yang Seagle Liu (Veracyte, Inc.)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Modern genomic tests in cancer treatment (products like Decipher for prostate and bladder cancer; Afirma for thyriod cancer), has led to the collection of genomic profiles of a large number of patients (N > 100K). This data has become a rich resource to improve existing genomic classifiers, but it is often impractical to obtain outcome data on such a large number of patients. To leverage this large unlabelled data, we explore semi-supervised machine learning approaches that utilize both labelled and unlabelled to train a machine learning model. This presentation is an overview of this data as well as approaches we explored.
            </td>
        </tr>
    </table>

    <table>

        <tr>
            <td class="date" rowspan="2">
                5:30 - 7:30 pm
            </td>
            <td class="title-special">
                Dinner
            </td>
        </tr>
        <tr>
            <td class="abstract">
                BBQ provided for everyone. Located in David Turpin Building first floor hallway. 
            </td>
        </tr>
    </table>


    <table>
        <tr>
            <td class="date" rowspan="2">
                6:00 - 9:00 pm
            </td>
            <td class="title-special">
                Poster Session
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Located in David Turpin Building first floor hallway.
            </td>
        </tr>
    </table>

<br>
<div style="width: 100%; height: 10px; border-bottom: 1px solid black; text-align: center">
  <span style="font-size: 24px; background-color: #fafafa; padding: 0 10px;">
    August 16<!--Padding is optional-->
  </span>
</div>
<br>

    <table>
        <tr>
            <td class="date" rowspan="2">
                9:20 am
            </td>
            <td class="title-special">
                Light Breakfast
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Breakfast provided for everyone. Located in the conference room.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
		9:45 am
            </td>
            <td class="title">
                Robust Joint Modelling of Left-Censored Longitudinal Data and Survival Data, with Application to HIV Vaccine Studies
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Lang Wu (University of British Columbia - Department of Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                In jointly modelling longitudinal and survival data, the longitudinal data may be complex in the sense that they may contain outliers and may be left censored. Motivated from an HIV vaccine study, we propose a robust method for joint models of longitudinal and survival data, where the outliers in longitudinal data are addressed using a multivariate t-distribution for b-outliers and using an M-estimator for e-outliers. We also propose a computationally efficient method for approximate likelihood inference. The proposed method is evaluated by simulation studies. Based on the proposed models and method, we analyze the HIV vaccine data and find a strong association between longitudinal biomarkers and the risk of HIV infection.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                10:25 am
            </td>
            <td class="title">
                Establishing LOD and LOQ for samples with low copy number eDNA
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Mary Lesperance (University of Victoria - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Quantitative real-time polymerase reaction (qPCR) is a popular, highly sensitive means to detect environmental DNA (eDNA) in a variety of sample matrices. Currently, there is a drive to use qPCR data to infer species biomass or abundance by quantifying the copy number or concentration of a given target gene fragment in a sample, which is often very dilute. Cycle thresholds (C<sub>t</sub>/C<sub>q</sub>) on multiple technical replicates have been used to quantify eDNA amounts. However, quantification of DNA copy number has been challenging when DNA is not detected in all technical replicates. Herein, we provide a statistically robust Binomial-Poisson model to create a standard curve that relates the number of qPCR-detected technical replicates to the copy number to be applied in the case of low copy number samples. Limits of detection (LOD) and quantification (LOQ) and their confidence intervals are derived using a well-accepted statistical approach thus providing a more broadly applicable and robust method for reporting eDNA abundance in the low copy number range. To date, we have applied this approach to 30+ eDNA assays from multiple labs. In this presentation, we provide a practical example on how to derive LOD and LOQ with confidence intervals and estimate copy numbers with standard errors using a standardized format and synthetic DNA to characterize an eDNA assay.
            </td>
        </tr>
    </table>

    <table>

        <tr>
            <td class="date" rowspan="2">
                11:05 am
            </td>
            <td class="title-special">
                Coffee Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Coffee break provided for everyone.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
            11:20 am   
            </td>
            <td class="title">
                Residual diagnostics for censored regression via randomized survival probabilities
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Longhai Li (University of Saskatchewan - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Residuals in normal regression are used to assess a model's goodness-of-fit (GOF) and discover directions for improving the model. However, there is a lack of residuals with a characterized reference distribution for censored regression. In this article, we propose to diagnose censored regression with normalized randomized survival probabilities (RSP). The key idea of RSP is to replace the survival probability (SP) of a censored failure time with a uniform random number between 0 and the SP of the censored time. We prove that RSPs always have the uniform distribution on (0, 1) under the true model with the true generating parameters. Therefore, we can transform RSPs into normally distributed residuals with the normal quantile function. We call such residuals normalized RSP (NRSP residuals). We conduct simulation studies to investigate the sizes and powers of statistical tests based on NRSP residuals in detecting the incorrect choice of distribution family and nonlinear effect in covariates. Our simulation studies show that, although the GOF tests with NRSP residuals are not as powerful as a traditional GOF test method, a nonlinear test based on NRSP residuals has significantly higher power in detecting nonlinearity. We also compared these model diagnostics methods with a breast-cancer recurrent-free time dataset. The results show that the NRSP residual diagnostics successfully captures a subtle nonlinear relationship in the dataset, which is not detected by the graphical diagnostics with CS residuals and existing GOF tests.
            </td>
        </tr>
    </table>

    <table>

        <tr>
            <td class="date" rowspan="2">
                12:00 pm
            </td>
            <td class="title-special">
                Lunch Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Lunch provided for speakers and volunteers. Located in Elliot Building room 165.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                2:00 pm
            </td>
            <td class="title">
                A novel machine learning approach for gene module identification and prediction via a co-expression network of single-cell sequencing data
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Li Xing (University of Saskatchewan - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Gene co-expression network (GCN) analysis is widely used in microarray and RNA sequencing data to group genes with correlated expression levels, inferring similarity in function or coregulation in the pathway. In literature, such approaches are mainly unsupervised, which may introduce instability and variation across different datasets. Inspired by modern machine learning, we propose a novel approach that integrates supervised and unsupervised learning and simultaneously targets two tasks for analysis of RNA sequencing data: (1) gene module identification; and (2) phenotype prediction.<br>
		The identified gene modules from this approach could enable other researchers to conduct follow-up studies to figure out causal genes. And the novel approach also improves the accuracy of predicting phenotypes. Its algorithm incorporates parallel computation and other strategies to ensure researchers can handle large-scale single-cell data using personal computers. We showcased its use for single-cell auto-annotation.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                2:40 pm
            </td>
            <td class="title">
                Performance of Cross-Fit and Doubly Robust Estimators for Residual Confounding Control in Pharmacoepidemiologic Studies Utilizing Administrative Healthcare Databases
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Ehsan Karim (University of British Columbia - School of Population and Public Health)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Retrospective health care claims datasets are not primarily collected for research purposes, and hence studies conducted using such databases are commonly criticized for the lack of complete information on potential confounders. A massive amount of diagnoses, procedures, and medication codes that are regularly captured in the claims databases are usually not used in a regular epidemiological study. In effect estimation studies, the high-dimensional propensity score (hdPS) algorithm is a framework that enables us to utilize such information as proxies of unobserved information and that has been shown to reduce bias. Some of the machine learning methods that can select or rank variables have been shown to perform as a suitable alternative to this hdPS framework. In this talk, using a health administrative cohort study as a motivating example, the performance of cross-fit and doubly robust estimators will be highlighted compared to both hdPS and machine learning methods, and practical recommendations will be discussed
            </td>
        </tr>
    </table>

    <table>

        <tr>
            <td class="date" rowspan="2">
                3:20 pm
            </td>
            <td class="title-special">
                Coffee Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Coffee break provided for everyone.
            </td>
        </tr>
    </table>

    <table id="FaroukNathoo">
        <tr>
            <td class="date" rowspan="3">
                3:50 pm
            </td>
            <td class="title">
                Neural Network Classifiers for Features Extraction in Neuroimaging Genetics 
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Farouk Nathoo (University of Victoria - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                A major issue in the association of genes to imaging phenotypes is the high dimension of both the genetic data and imaging data. In the article, we tackled the latter problem. Our proposed solution is supported by a vast literature and uses the great predictive power of neural networks to extract features highly related with Alzheimers Disease (AD) diagnosis. A key concept of our work is a neuroimaging genetic pipeline where we separate the neuroimaging genetic association in three distinct steps: first is image processing, then, neuroimaging feature extraction and finally the genetic association study. In this article, we discuss the use of a neural network classifier in order to accomplish the second step; extracting neuroimaging features that are related with AD. We compared the predictive power of those features to expert selected features before running a genetic association study and take a closer look at the genes identified given those new neuroimaging features.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                4:30 pm
            </td>
            <td class="title">
                Using CVX to construct optimal designs for biomedical studies with multiple objectives
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Julie Zhou (University of Victoria - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Model-based optimal designs for regression problems with multiple objectives are common in practice. The traditional approach is to construct an optimal design for the most important objective and hope that the design performs well for the other objectives. Analytical approaches are challenging because the objectives are often competitive and their relative importance has to be incorporated at the onset of the design construction. There are also no general and efficient algorithms for searching such designs for user-specified nonlinear models and criteria. We propose a new and effective approach for finding multiple-objective optimal designs via the CVX software and demonstrate it can efficiently find different types of multiple-objective optimal designs after the optimization problems are carefully formulated as convex optimization problems appropriate for CVX use. We provide biomedical applications and show our MATLAB codes work well.  This is a joint work with Professor Weng Kee Wong.
            </td>
        </tr>
    </table>

    <table>
        <tr>
            <td class="date" rowspan="2">
                6:30 - 9:30 pm
            </td>
            <td class="title-special">
                Dinner
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Dinner provided for speakers and volunteers. Located at <a href="https://1550srestaurant.com/">1550's Pub Style Restaurant</a>.
            </td>
        </tr>
    </table>

<br>
<div style="width: 100%; height: 10px; border-bottom: 1px solid black; text-align: center">
  <span style="font-size: 24px; background-color: #fafafa; padding: 0 10px;">
    August 17<!--Padding is optional-->
  </span>
</div>
<br>

    <table>
        <tr>
            <td class="date" rowspan="2">
                9:20 am
            </td>
            <td class="title-special">
                Light Breakfast
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Breakfast provided for everyone. Located in the conference room.
            </td>
        </tr>
    </table>
<!--
    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
		9:45 am
            </td>
            <td class="title">
                Ensembling Classification Models Based on Phalanxes of Variables with Applications in Drug Discovery
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Jabed Tomal (Thompson Riveres University - Department of Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Statistical detection of a rare class of objects in a two-class classification problem can pose several challenges. Because the class of interest is rare in the training data, there is relatively little information in the known class response labels for model building. At the same time the available explanatory variables are often moderately high dimensional. In the four assays of our drug-discovery application, compounds are active or not against a specific biological target, such as lung cancer tumor cells, and active compounds are rare. Several sets of chemical descriptor variables from computational chemistry are available to classify the active versus inactive class; each can have up to thousands of variables characterizing molecular structure of the compounds. The statistical challenge is to make use of the richness of the explanatory variables in the presence of scant response information. Our algorithm divides the explanatory variables into subsets adaptively and passes each subset to a base classifier. The various base classifiers are then ensembled to produce one model to rank new objects by their estimated probabilities of belonging to the rare class of interest. The essence of the algorithm is to choose the subsets such that variables in the same group work well together; we call such groups phalanxes.
            </td>
        </tr>
    </table>
-->

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                9:45 am
            </td>
            <td class="title">
                A Bayesian adaptive multilayer basis model for task fMRI data
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Michelle Miranda (University of Victoria - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Task-evoked functional magnetic resonance imaging (fMRI) studies are a powerful tool to understand human sensory, cognitive, and emotional processes. We introduce a new Bayesian approach for analyzing task fMRI data that simultaneously detects activation signatures and background connectivity. Our modeling involves a new tensor spatial-temporal basis strategy that enables scalable computing yet captures spatial correlation from nearby voxels and distant ROIs and long-memory temporal correlation.  The spatial basis involves a composite hybrid transform with two levels: the first accounts for within-ROI correlation, and second between-ROI distant correlation.  We demonstrate in simulations how our basis space regression modeling strategy increases sensitivity for identifying activation signatures, partly driven by the induced background connectivity that itself can be summarized to reveal biological insights.  This strategy leads to computationally scalable fully Bayesian inference at the voxel or ROI level that adjusts for multiple testing.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                10:25 am
            </td>
            <td class="title">
                Bayesian Adaptive Dose-Finding Design in Phase I/II trial
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Haolun Shi (Simon Fraser University - Department of Statistics and Actuarial Science)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                MMolecularly targeted agents and immunotherapy have revolutionized modern cancer treatment. Unlike chemotherapy, the maximum tolerated dose of the targeted therapies may not pose significant clinical benefit over the lower doses. By simultaneously considering both binary toxicity and efficacy endpoints, phase I/II trials can identify a better dose for subsequent phase II trials than traditional phase I trials in terms of efficacy-toxicity tradeoff.  Existing phase I/II dose-finding methods are model-based or need to pre-specify many design parameters, which makes them difficult to implement in practice. To strengthen and simplify the current practice of phase I/II trials, we propose a utility-based toxicity probability interval (uTPI) design for finding the optimal biological dose (OBD) where binary toxicity and efficacy endpoints are observed. The uTPI design is model-assisted in nature, simply modeling the utility outcomes observed at the current dose level based on a quasibinomial likelihood. Toxicity probability intervals are used to screen out overly toxic dose levels, and then the dose escalation/de-escalation decisions are made adaptively by comparing the posterior utility distributions of the adjacent levels of the current dose. The uTPI design is flexible in accommodating various utility functions while only needs minimum design parameters. A prominent feature of the uTPI design is that it has a simple decision structure such that a concise dose-assignment decision table can be calculated before the start of trial and be used throughout the trial, which greatly simplifies practical implementation of the design. Extensive simulation studies demonstrate that the proposed uTPI design yields desirable as well as robust performance under various scenarios.
            </td>
        </tr>
    </table>

    <table>

        <tr>
            <td class="date" rowspan="2">
                11:05 am
            </td>
            <td class="title-special">
                Coffee Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Coffee break provided for everyone.
            </td>
        </tr>
    </table>


<!--
    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                
            </td>
            <td class="title">
                Title TBD 
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Gabriela V. Cohen Freue (University of British Columbia - Department of Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Abstract TBD
            </td>
        </tr>
    </table>
-->
<!---
    <table>
        <tr>
            <td class="date" rowspan="2">
                10:40am
            </td>
            <td class="title-special">
                Coffee &amp; Tea Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                 Kept for Spacing
            </td>
        </tr>
    </table>
-->
<!---
    <table class="plenary">
        <tr>
            <td class="date" rowspan="3">
                11:00am
            </td>
            <td class="title">
                This is a Plenary Talk!
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Plenary Speaker
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Here&rsquo;s the abstract for the plenary talk! Notice again that the formatting of this time-block is a bit different that the rest of of the talks.
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>
<!---
    <table>
        <tr>
            <td class="date" rowspan="2">
                Noon
            </td>
            <td class="title-special">
                Lunch
            </td>
        </tr>
        <tr>
            <td class="abstract">
                 Kept for Spacing 
            </td>
        </tr>
    </table>
-->
 


 



<!---
    <table>
        <tr>
            <td class="date" rowspan="2">
                2:50pm
            </td>
            <td class="title-special">
                Coffee &amp; Tea Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                <!-- Kept for Spacing -->
            </td>
        </tr>
    </table>







<!---
    <table class="plenary">
        <tr>
            <td class="date" rowspan="3">
                4:00pm
            </td>
            <td class="title">
                This is Another Plenary Talk!
            </td>
        </tr>
        <tr>
            <td class="speaker">
                Plenary Speaker
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Filler text: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum dignissim et est et euismod. Fusce et metus tempus, pellentesque ex at, convallis nulla. Ut fringilla commodo tincidunt. Fusce sed est eu massa placerat iaculis eu at mauris. Nullam ut mollis nisi, quis malesuada risus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Nam ipsum tortor, suscipit non tincidunt vel, bibendum in libero. Nulla facilisi. Pellentesque vitae neque metus. Cras quis est pharetra, vestibulum nisl et, viverra ipsum. Etiam porta dignissim purus, quis tempor metus volutpat eu. Praesent pulvinar libero eget purus tincidunt finibus.
            </td>
        </tr>
    </table>
--->
















    <footer>
        &copy; Conference Organizers
        &nbsp;|&nbsp; Design by <a href="https://github.com/mikepierce">Mike Pierce</a>
    </footer>

</body>
</html>

